specification/containerregistry/data-plane/Azure.ContainerRegistry/preview/2019-08-15-preview/containerregistry.json

```diff
@@ -1,3091 +1,3091 @@
+                  "description": "Specifies a list of URIs from which this object may be downloaded.",
+                  "items": {
+                      "description": "Must conform to RFC 3986. Entries should use the http and https schemes, as defined in RFC 7230.",
+                      "type": "string"
+                  }
+              },
+              "annotations": {
+                  "$ref": "#/definitions/Annotations"
+              }
           },
-          "default": {
-            "description": "ACR error response describing why the operation failed.",
-            "schema": {
-              "$ref": "#/definitions/AcrErrors"
-            }
+          "example": {
+              "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
+              "size": 2107098,
+              "digest": "sha256:5d20c808ce198565ff70b3ed23a991dd49afac45dece63474b27ce6ed036adc6"
           }
-        }
-      }
-    }
-  },
-  "definitions": {
-    "RepositoryAttributes": {
-      "required": [
-        "registry",
-        "imageName",
-        "createdTime",
-        "lastUpdateTime",
-        "manifestCount",
-        "tagCount",
-        "changeableAttributes"
-      ],
-      "x-ms-client-name": "ContainerRepositoryProperties",
-      "type": "object",
-      "description": "Properties of this repository.",
-      "properties": {
-        "registry": {
-          "type": "string",
-          "readOnly": true,
-          "description": "Registry login server.",
-          "x-ms-client-name": "registryLoginServer"
-        },
-        "imageName": {
-          "type": "string",
-          "readOnly": true,
-          "description": "Image name",
-          "x-ms-client-name": "name"
-        },
-        "createdTime": {
-          "type": "string",
-          "readOnly": true,
-          "format": "date-time",
-          "description": "Image created time",
-          "x-ms-client-name": "createdOn"
-        },
-        "lastUpdateTime": {
-          "type": "string",
-          "readOnly": true,
-          "format": "date-time",
-          "description": "Image last update time",
-          "x-ms-client-name": "lastUpdatedOn"
-        },
-        "manifestCount": {
-          "type": "integer",
-          "readOnly": true,
-          "description": "Number of the manifests"
-        },
-        "tagCount": {
-          "type": "integer",
-          "readOnly": true,
-          "description": "Number of the tags"
-        },
-        "changeableAttributes": {
-          "description": "Writeable properties of the resource",
-          "$ref": "#/definitions/RepositoryChangeableAttributes",
-          "x-ms-client-flatten": true
-        }
-      },
-      "example": {
-        "registry": "registryname.azurecr.io",
-        "changeableAttributes": {
-          "readEnabled": true,
-          "listEnabled": true,
-          "deleteEnabled": true,
-          "writeEnabled": true,
-          "teleportEnabled": false
```

Comment:
I'm assuming this will be removed from the GA swagger.

Reply:
Yes, it's removed